lib/tasks/db.rake

```diff
@@ -0,0 +1,46 @@
+namespace :db do
+
+  desc 'Dumps the database to file'
+  task :dump => :environment do
+
+    host = ActiveRecord::Base.connection_config[:host]
+    db = ActiveRecord::Base.connection_config[:database]
+
+    cmd = "pg_dump --no-owner --no-acl -v -c -f #{dump_file} --no-password "
+    cmd += ENV['DATABASE_URL'] || "--host #{host} --dbname #{db}"
```

Comment:
If a `DATABASE_URL` is not present, specify _just_ the `host` would not be sufficient to establish a connection. You would likely need to form a URL similar to the format of `DATABASE_URL`, which is:

`postgres://username:pass@host:port/dbname`